python/libkvikio/pyproject.toml

```
@@ -0,0 +1,48 @@
+# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+# See file LICENSE for terms.
+
+[build-system]
+build-backend = "scikit_build_core.build"
+requires = [
+    "cmake>=3.26.4",
+    "ninja",
+    "scikit-build-core[pyproject]>=0.7.0",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[project]
+name = "libkvikio"
+dynamic = ["version"]
+description = "KvikIO - GPUDirect Storage"
```

```suggestion
description = "KvikIO - GPUDirect Storage (header-only library)"
```

I noticed this description is identical to the one for the `kvikio` wheel. Would you consider adding something like this to differentiate them?

*(this is super minor, so only worry about it if any of the other review comments lead to more commits... this is not worth a new CI cycle)*

---

I don't think marking it as header-only is what we want. I would prefer to document that it's the C++ package (vs the Python package). We can do that consistently across RAPIDS. I wouldn't mention header-only vs not in this short description.

---

Makes sense! I figured something like that was why we're pursuing wheel names like `lib{project}` instead of `lib{project}-header-only`.

```suggestion
description = "KvikIO - GPUDirect Storage (C++)"
```